### PR TITLE
Create a CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by rolfedh**
to comply with the guidelines for adding new content in https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @rolfedh

This pull request needs review by: I'm not sure.

### CODE REVIEW

**Comment by rolfedh** [comment]
> This is in contradiction with usual naming practice, with the work in #2071 too. The documentation needs to be updated the other way around. We can either do this in this PR or I can update #2071.

Thanks, @duck-rh. I've updated the file name. If this is correct, we should also update the following line in [How to Contribute](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): "From your local Git repo folder, navigate to the relevant directory, create a new text file and save it with the following naming convention: <name>.html.md"

**Comment by duck-rh** [comment]
@rolfedh there were a few more references I missed so I updated my PR and I'm thus closing yours. Thanks for pointing this out.

**Comment by rolfedh** [comment]
> @rolfedh there were a few more references I missed so I updated my PR and I'm thus closing yours. Thanks for pointing this out.

@duck-rh Sorry I wasn't clear in my original pull request name.  I've revised it to be clearer. The intention of this pull request is to develop a code of conduct for the oVirt project. This idea was suggested during Brian Profitt's "Community Central: Introducing the OSS Participation Guidelines" event at Red Hat this week. This is certainly something for the body of contributors to review and discuss before adoption. This initial version of the code of conduct comes from the template proposed by GitHub itself. The content seems pretty clear and reasonable.

**Comment by duck-rh** [comment]
@rolfedh sorry I misread. In fact I was sure we already had one as this was discussed in a team meeting long ago. So, thanks for making this happen :-)

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at TBD@example.com. All
```
Comment (rolfedh) [inline CODE_OF_CONDUCT.md]: @duck-rh What is the best way to report issues to the project team? We should update this line with that information.

Comment (rolfedh) [reply CODE_OF_CONDUCT.md]: Closing this pull request. I found our code of conduct somewhere else.
https://ovirt.org/community/about/community-guidelines.html

**Comment by rolfedh** [comment]
Closing this pull request. I found our code of conduct somewhere else.
https://ovirt.org/community/about/community-guidelines.html

**Pull request closed without merging**